src/components/App/App.js

```diff
@@ -63,7 +63,7 @@ function App() {
         });
       })
       .catch(error => {
-        return new Error(error);
+        throw new Error(error);
```

Comment:
`getFarmersHandler` function is throwing an error if the axios request fails in its `catch` block. In this `catch` block we should do something with that error to let the user know the request failed, so I'm thinking instead of throwing the error again, we can display an `toast.error` with some message.